### ISSUE TITLE
Solver crashes instead of reverting to MUMPS

### ISSUE THREAD

**Issue by jamesmurdza** [Libbum/DICE.jl#44]
I can’t tell if this is an existing issue or one I have introduced in my recent changes. I’m sure that installing the appropriate libraries would fix the issue, however it should not be crashing.

**Works:**

`dice = solve(BasePrice, version, linear_solver = mumps);`

 **Fails:**

`base = solve(BasePrice, version)`

```
Exception of type: DYNAMIC_LIBRARY_FAILURE in file "Common/IpLibraryLoader.cpp" at line 67:
 Exception message: libhsl.so: cannot open shared object file: No such file or directory

EXIT: Library loading failure.
Exception of type: DYNAMIC_LIBRARY_FAILURE in file "Common/IpLibraryLoader.cpp" at line 67:
 Exception message: libhsl.so: cannot open shared object file: No such file or directory

EXIT: Library loading failure.
```

**Comment by Libbum** [comment]
Thanks for the changes! Yep, I'll take a look and see, but the expectation is that not finding hsl would fallback to mumps by default. This shouldn't happen.

**Comment by Libbum** [comment]
Fixed in #45
